libs/gl-client/src/lnurl/mod.rs

```diff
@@ -0,0 +1,15 @@
+mod pay;
+mod tor_http_client;
+
+use anyhow::{Result};
+use pay::resolve_to_invoice;
+use pay::models::{LnUrlHttpClientImpl};
+
+pub struct LNURL;
```

Comment:
Probably will need to have access to a `ClnClient` so it can trigger operations on the `Node`.

Reply:
Discussed in-person: keeping it exploded for now, asking the user to ferry information back and forth, allowing to hook into the process makes sense. We can always build a `SimpleLnUrl` struct that wraps the node and the LNURL struct to simplify its use for users.